addons/mail/static/src/core/common/composer.js

```diff
@@ -628,10 +578,7 @@ export class Composer extends Component {
         const composer = toRaw(this.props.composer);
         if (composer.text || composer.message.attachments.length > 0) {
             await this.processMessage(async (value) =>
-                composer.message.edit(value, composer.attachments, {
-                    mentionedChannels: composer.mentionedChannels,
-                    mentionedPartners: composer.mentionedPartners,
-                })
+                composer.message.edit(value, composer.attachments, composer.mentions)
```

Comment:
composer.mentions can be undefined or not?

Reply:
I believe this relational field can't be undefined but I might be mistaken